UnitTests/ClientTest/ModClient.h

```diff
@@ -45,7 +44,7 @@ class ModClient final : public vasily::Client
 	/**
 	 * \brief Array of sent data to server to check.
 	 */
-	std::vector<std::string>	storage;
+	std::vector<std::string>	storage{};
```

Comment:
Why you use {} here?

Reply:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Pro-type-memberinit

Type.6: Always initialize a member variable: always initialize, possibly using default constructors or default member initializers.

Reply:
Do it in constructor, not in declaration.Wha not?

Reply:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c48-prefer-in-class-initializers-to-member-initializers-in-constructors-for-constant-initializers

Reply:
Amm... Please, read links that I send to you.